config/clusters/hhmi/spyglass.values.yaml

```
@@ -72,12 +87,9 @@ jupyterhub:
   hub:
     config:
       JupyterHub:
-        authenticator_class: cilogon
-      CILogonOAuthenticator:
-        oauth_callback_url: https://spyglass.hhmi.2i2c.cloud/hub/oauth_callback
-        allowed_idps:
-          http://github.com/login/oauth/authorize:
-            default: true
-            username_derivation:
-              username_claim: "preferred_username"
-            allow_all: true
+        authenticator_class: tmpauthenticator.TmpAuthenticator
```

```suggestion
        authenticator_class: tmp
```

Added in tmpauthenticator 1.0.0 via https://github.com/jupyterhub/tmpauthenticator/commit/f2f6deed9962223a7386b53d2ae5c98844195f06

---

Done in https://github.com/2i2c-org/infrastructure/pull/3669